src/regtech_user_fi_management/entities/repos/institutions_repo.py

```diff
@@ -104,34 +87,25 @@ async def update_sbl_types(
         fi.sbl_institution_types.extend(add_types)
         for type in fi.sbl_institution_types:
             type.version = fi.version
-        await session.commit()
-        """
-        load the async relational attributes so dto can be properly serialized
-        """
-        for type in fi.sbl_institution_types:
-            await type.awaitable_attrs.sbl_type
```

Comment:
what was the purpose of this?  Just forcing a reload of relationship data in the model? Seemed unnecessary? Good cleanup!

Reply:
it was because the relational model was another nest deeper, so it couldn't populate the 2nd layer of nested relationship if I recall correctly, so awaitable_attrs was needed; the standard apis don't have this concern during my testing.